brigade/templates/index.html

```diff
@@ -20,7 +20,7 @@
         <div id="map"></div>
       </div>
       <div class="panel__footer">
-        Looking for an organization outside the US? <a href="https://codeforall.org/" target="_blank" class="one-line">Check out the <strong>Code for All</strong> Network</a>
+        Looking for an organization outside the US? <a href="https://codeforall.org/" target="_blank" class="inline-block">Check out the <strong>Code for All</strong> Network</a>
```

Comment:
If the intent of this helper class is to avoid breaking the link's text between multiple lines, I think a more semantic way would be to use `white-space: nowrap`. Do you know of differences between that approach and this one?

Reply:
You're right! Good call, will change it out.